pmd-cli/src/main/java/net/sourceforge/pmd/cli/commands/internal/PmdCommand.java

```diff
@@ -323,6 +323,16 @@ protected ExecutionResult execute() {
                 }
 
                 pmdReporter.log(Level.DEBUG, "Current classpath:\n{0}", System.getProperty("java.class.path"));
```

Comment:
These `log` calls with level debug or trace should be changed to calls on a Logger instance defined here (or eg in PmdCli). The MessageReporter interface is supposed to be used only with levels info and up. #3816

Reply:
Done: https://github.com/pmd/pmd/commit/0ad3755a2b128c9c56fd7f28975dd90adbde8880